### PR TITLE
init repo

### PR DESCRIPTION
**Pull request by t-rutten**

### CODE REVIEW

**Comment by t-rutten** [comment]
@josevalim do you want to go with SciData or Scidata in module names?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# SciData
```
Comment (josevalim) [inline README.md]: Sorry for the nitpick. If we want to call it `SciData` then the convention is to call the application (and the repository) `sci_data` - with underscore. We can easily adjust it though. Which one do you prefer? :)

**Comment by josevalim** [comment]
Looks great to me, I just have one very small nitpick. :) Let me know what you think.

File: README.md (hunk 1 above)
Comment (t-rutten) [reply README.md]: I think `Scidata` is good! I'll change the module names.

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**